erts/config.h.in

```diff
@@ -438,6 +438,9 @@
 /* Define if x86/x86_64 out of order instructions should be synchronized */
 #undef ETHR_X86_OUT_OF_ORDER
 
+/* Define to 1 if _Float16 can be converted from/to double. */
+#undef FLOAT16_IS_CONVERTIBLE
```

Comment:
Changes related to FLOAT16 are not part of this PR, but were added when I ran `otp_build update_configure`

Reply:
Someone has merged a `configure.ac` fix without also merging the results from `otp_build update_configure`. If you base your fix on a released OTP version, that will not happen as our release procedure ensures that `otp_build update_configure` results have been merged.

Reply:
Yep, that's what I figured. If you'd like to remove it from this PR, feel free, or I can.

Reply:
If you don't want to rebase it, you can leave it as is. The information of what you did is still valuable to the person taking care of the related issue.